src/launcher/java/org/truffleruby/launcher/Metrics.java

```diff
@@ -0,0 +1,49 @@
+package org.truffleruby.launcher;
+
+import org.graalvm.launcher.Launcher;
+
+import java.lang.management.ManagementFactory;
+
+public class Metrics {
+
+    // These system properties are used before outside the SDK option system
+    public static boolean METRICS_TIME;
+    private static final boolean METRICS_MEMORY_USED_ON_EXIT =
+            Boolean.getBoolean("truffleruby.metrics.memory_used_on_exit");
+
+    public static void printTruffleTime(String id) {
```

Comment:
Minor: the `Truffle` part in this method name dates back from JRuby I think, so we could simplify to just `printTime()` now.

Reply:
done